src/ray/gcs/gcs_server/gcs_monitor_server.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "src/ray/protobuf/monitor.pb.h"
+#include "src/ray/rpc/gcs_server/gcs_rpc_server.h"
+
+namespace ray {
+namespace gcs {
+
+/// GcsNodeManager is responsible for managing and monitoring nodes as well as handing
```

Comment:
replace `GcsNodeManager` with `GcsMonitorServer `

Reply:
And the rest of the docstring too I suppose